packages/astro/src/core/util.ts

```diff
@@ -243,8 +243,8 @@ export function resolveJsToTs(filePath: string) {
 }
 
 export const AggregateError =
-	typeof globalThis.AggregateError !== 'undefined'
-		? globalThis.AggregateError
+	typeof (globalThis as any).AggregateError !== 'undefined'
+		? (globalThis as any).AggregateError
```

Comment:
I believe the `type-fest` upgrade made this fail. I suspect we were depending on it by accident, weird